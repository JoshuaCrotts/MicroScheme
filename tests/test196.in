;
; Harder complex number math
;

(define c1 18+9i)
(asin (acos (+ c1 3.125-11.9i)))
