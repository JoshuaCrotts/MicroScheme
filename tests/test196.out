-0.0351558170965421721216064-2.03344251295472101860312i
